packages/x-tree-view/src/internals/plugins/useTreeViewItems/useTreeViewItems.selectors.ts

```diff
@@ -7,16 +7,6 @@ import { UseTreeViewItemsSignature } from './useTreeViewItems.types';
 const EMPTY_CHILDREN: TreeViewItemId[] = [];
 
 export const itemsSelectors = {
-  /**
```

Comment:
I moved the root tree view loading and error states in the lazy loading plugin just like the nested ones.
That way we don't have two paths to access similar data